docs/gitbook/key-concepts/hypervisor.adoc

```diff
@@ -0,0 +1,4 @@
+= Hypervisor
+
+The small piece of code that boots interbit and allows hosting of
```

Comment:
Where did we get this definition from?

Reply:
I don't know. It was already in the documentation; I just split it out into its own file so that we have a good location for future elaboration.

Reply:
ah okay, fair enough then :)